spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,150 @@
+RSpec.describe UsersController, type: :controller do
+  let(:user) { FactoryBot.create(:user) }
+  let(:another_user) { FactoryBot.create(:user) }
+
+  describe 'GET #index' do
+    before { get :index }
+
+    it { is_expected.to respond_with(:success) }
+    it { is_expected.to render_template(:index) }
+
+    it 'assigns all users to @users' do
+      user
+      another_user
```

Comment:
can you use let! ?

Reply:
Bhai, I used let! for :user because :user is used by many test, and used let for :another_user because :another_user is used by  only one test.